pyVMC.py

```diff
@@ -180,70 +209,111 @@ def getCSPGroupDiff(csp_url, session_token):
+            sys.exit(1)
 
-    if len(sys.argv) < 3:
-        print('Usage: show-csp-groups [GROUP_SEARCH_TERM]. You must include a search term with characters contained in the group name. No globbing allowed.')
-    else:
-        searchTerm = sys.argv[2] 
-        json_response = get_csp_groups_json(csp_url, ORG_ID, session_token,searchTerm)
-        if json_response != None:
-            groups = json_response['results']
-            numGroups = len(groups)
-            if(numGroups == 0):
-                print("No results returned.")
-            else:
-                print(str(numGroups) + " result" + ("s" if numGroups > 1 else "") + " returned:")
-                table = PrettyTable(['ID','Name', 'Group Type','User Count'])
-                for grp in groups:
-                    table.add_row([grp['id'],grp['displayName'], grp['groupType'], grp['usersCount']])
-                print(table)
+    except:
+        json_response = get_csp_groups_json(strCSPProdURL, ORG_ID, sessiontoken)
+        if json_response == None:
+            print("API Error")
+            sys.exit(1)
 
+    if json_response is not None:
+        groups = json_response['results']
+        numGroups = len(groups)
+        if(numGroups == 0):
+            print("No results returned.")
+        else:
+            print(str(numGroups) + " result" + ("s" if numGroups > 1 else "") + " returned:")
+            table = PrettyTable(['ID','Name', 'Group Type','User Count'])
+            for grp in groups:
+                table.add_row([grp['id'],grp['displayName'], grp['groupType'], grp['usersCount']])
+            print(table)
 
-def getCSPOrgUsers(csp_url,session_token):
-    if len(sys.argv) < 3:
-        print('Usage: show-csp-org-users [searchTerms]')
+
+def searchCSPOrgUsers(**kwargs):
+    # for i, j in kwargs.items():
+    #     print(i, j)
+    sessiontoken = kwargs['sessiontoken']
+    ORG_ID = kwargs['ORG_ID']
+    strCSPProdURL = kwargs['strCSPProdURL']  
+    if kwargs['search_term'] is None:
+        print("Plese enter a search term (--search-term).  To simply show all ORG users, please use show-org-users")
+        sys.exit()
     else:
-        searchTerm = sys.argv[2]
-        params = {
+        searchTerm = kwargs['search_term']
+    params = {
             'userSearchTerm': searchTerm
         }
-        json_response = search_csp_users_json(csp_url, session_token, params, ORG_ID)
-        users = json_response['results']
-        if len(users) >= 20:
-            print("Search API is limited to 20 results, refine your search term for accurate results.")
-        table = PrettyTable(['Username', 'First Name', 'Last Name', 'Email', 'userId'])
-        for user in users:
-            table.add_row([user['user']['username'], user['user']['firstName'], user['user']['lastName'], user['user']['email'], user['user']['userId']])
-        print(table)
+    json_response = search_csp_users_json(strCSPProdURL, sessiontoken, params, ORG_ID)
```

Comment:
Double check the API we're using here.  There is a v2 that returns more than 20 results